feat(mtctl): initial version (#39)
tailor-platform/frontend-packages: packages/minitailorctl/README.md

# minitailorctl

> minitailorctl (mtctl) is a CLI tool that operates minitailor and contains some useful features that help development of Tailor Platform application.

Reviewer: "contains some useful features" isn't really helpful, how about something like

```suggestion
minitailorctl (mtctl) is a CLI tool that wraps [tailorctl](https://github.com/tailor-platform/tailorctl) to streamline the development of Tailor Platform applications.
```

## Install

```bash
npm install -g @tailor-platform/minitailorctl
```

> Then, `minitailorctl` command (and `mtctl` as alias) will be available on your machine.

Reviewer: ```suggestion
As a result, the `minitailorctl` command (and its `mtctl` alias) will be available in your terminal.
```

> 

Reviewer: I think around here we need to have a link or a short mention about minitailor and how to install/run it. Without it this package is useless and it'd be nice to have newcomers have all the info/relevant links here.

Author: I am going to add feature in minitailorctl later on to install and run minitailor, so I would like to skip mentioning it in this PR.

## Features

### Tailorctl proxy

> minitailorctl can run any commands on [tailorctl](https://github.com/tailor-platform/tailorctl) without explictly configured your envivonment to connect it to minitailor.

Reviewer: ```suggestion
minitailorctl can run any command from [tailorctl](https://github.com/tailor-platform/tailorctl) and is pre-configured to communicate with a locally running minitailor instance. 
```

```bash
# The below is the same as "tailorctl alpha workspace create --name test-workspace",
# but always runs with environment variables to connect to minitailor internally.

> $ minitailorctl -- alpha workspace create --name test-workspace
> Workspace test-workspace created

Reviewer: to make the highlighting work better how about

```suggestion
minitailorctl -- alpha workspace create --name test-workspace
# => Workspace test-workspace created
```
